addons/dragging/functions/fnc_dropObject_carry.sqf

```diff
@@ -30,8 +30,17 @@ private _inBuilding = _unit call FUNC(isObjectOnObject);
 [QEGVAR(common,fixCollision), _unit] call CBA_fnc_localEvent;
 [QEGVAR(common,fixCollision), _target, _target] call CBA_fnc_targetEvent;
 
-// Release object
-detach _target;
+getCursorObjectParams params ["_cursorObject", "", "_distance"];
```

Comment:
I don't know how I missed this, but you can load items into a vehicle, when your back is turned to it.
`getCursorObjectParams` acts like `cursorTarget`, meaning it's very broad in its selection. It means also that you can't place a cargo item next to a vehicle, cause it will automatically give you the load option.

I was thinking of switching to the `cursorObject` command, but using `getCursorObjectParams select 2` for the distance. This would allow for finer control.

Thoughts?

Reply:
At that point why not just do `ACE_player distance cursorObject` ?

Reply:
> At that point why not just do `ACE_player distance cursorObject` ?

Could do - but do you agree with the change in principle?

Reply:
Yes

Reply:
Implemented - can you review the most recent changes again?